app/src/main/java/org/kiwix/kiwixmobile/main/KiwixWebViewClient.java

```diff
@@ -109,8 +114,21 @@ private void inflateHomeView(WebView view) {
     home = inflater.inflate(R.layout.content_main, view, false);
     callback.setHomePage(home);
     if (sharedPreferenceUtil.nightMode()) {
+      ImageView sideImage = home.findViewById(R.id.content_side_image);
       ImageView cardImage = home.findViewById(R.id.content_main_card_image);
-      cardImage.setImageResource(R.drawable.ic_home_kiwix_banner_night);
+      AppCompatButton downloadButton = home.findViewById(R.id.content_main_card_download_button);
```

Comment:
Use ButterKnife to inject views.

Reply:
@abdulwd Injection for views done. Injection for color could not be done as there is no documentation of resources binding for non-activity.